examples/linux64/file-reading/example.txt

```diff
@@ -0,0 +1,6 @@
+The PPCI (Pure Python Compiler Infrastructure) project is a compiler
```

Comment:
This appears a more generic description of the project.

It is better to:
- replace this `example.txt` with a `README.md`, which will show up in github
- replace this text with a description of the example